x-pack/plugins/cloud/public/components/chat/chat.stories.tsx

```diff
@@ -75,6 +94,13 @@ export const Component = () => {
         </EuiFlexItem>
       </EuiFlexGroup>
       <Chat onHide={() => setIsHidden(true)} />
-    </>
+    </ServicesProvider>
   );
 };
+
+Component.args = {
+  id: '1234567890',
+  email: 'email.address@elasticsearch.com',
+  chatURL: 'https://elasticcloud-production-chat-us-east-1.s3.amazonaws.com/drift-iframe.html',
+  jwt: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwiaWF0IjoxNTE2MjM5MDIyfQ.L8i6g3PfcHlioHCCPURC9pmXT7gdJpx3kOoyAfNUwCc',
```

Comment:
I trust that this isn't anything sensitive 😉

Reply:
Shouldn't be... but now you have me wondering. :-)